apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -1388,8 +1388,6 @@ func (s *modelManagerStateSuite) TestAdminCanCreateModelForSomeoneElse(c *gc.C)
 	newModel, err := newState.Model()
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(newModel.Owner(), gc.Equals, owner)
-	_, err = newState.UserAccess(owner, newModel.ModelTag())
```

Comment:
**todo:** Removing this line actually change the test description above:

```go
	// Make sure that the environment created does actually have the correct
	// owner, and that owner is actually allowed to use the environment.
```
=> The test doesn't check that the owner is actually allowed to use the environment.

**question:** Is it ok to just remove the test without replacing it by something else ? (I miss the context sorry if it is a dumb question)

Reply:
Good point. I have deleted the comment.

So the reason this is fine to delete is because these tests are transitioning from using a real mongo as state, to using mocked calls. In the above function `expectCreateModelStateSuite` the calls to the domain are all expected with go-mock. So the admin user is never actually in any real state, we just get a call to   the domain CreateModel method that should be doing that. Since we have that, there is nothing more to test.